images/installer-origin-release/Dockerfile.ci

```diff
@@ -0,0 +1,12 @@
+# This Dockerfile is used as a CI job to build smoke tests for OpenShift installer
+
+FROM  quay.io/coreos/origin-release:bazel
+WORKDIR /go/src/github.com/openshift/installer
+COPY . .
+
+ENV USER="bazel"
+ENV HOME="/tmp"
+
+RUN ls -al && \
+    /usr/bin/bazel --output_base=/tmp build smoke_tests && \
```

Comment:
I expect `/usr/bin` is in the `PATH`, in which case I'd rather just have `bazel` here (like you have `cp` below).

Reply:
yup, thanks